app/frontend/src/components/edit-panel.js

```diff
@@ -32,12 +32,12 @@ function title(editMode) {
 }
 
 export const EditPanel =
-  inject("mainStore")(
-    observer(({ mainStore }) =>
+  inject("mainStore", "permissionsStore")(
+    observer(({ mainStore, permissionsStore }) =>
       <SidePanel
         title={title(mainStore.editMode)}
         opened={mainStore.editMode !== EditMode.None}
-        onClose={() => { mainStore.editMode = EditMode.None; mainStore.fileUploadIsOpen = false; }}
+        onClose={() => { mainStore.editMode = EditMode.None; mainStore.fileUploadIsOpen = false; permissionsStore.permissionsChanged = true; }}
```

Comment:
Why are we setting `permissionsStore.permissionsChanged` to `true` when a user closes the panel?